### PR TITLE
Rs 2021 11 23

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
-Us just going out looking for him and defeating him? That's not even worth considering.
-That kind of thinking is just an irresponsible escape from reality to ease my own guilt.
+Tracking him down and defeating him as we are right now? It's not even worth considering.
+That kind of wishful thinking is just an irresponsible escape from reality to ease my own guilt.
 <Page3>
 ...I need to collect myself.
-I must do so if I really want to defeat that vampire.
-I understand that if I truly want to reduce the number of casualties, I have no choice but to wait here.
+I have to if I really want to defeat that vampire.
+I understand that if I truly wish to reduce the number of casualties, I've no choice but to wait here.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:11]: this is a British-ism, "I have no choice" sounds a lot more natural

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 2]
```diff
@@ -3,22 +3,22 @@
...
 <Page4>
 But still―――three days is far too long.
 <Page5>
 "...Arcueid.
 You said that my duty as a guard ended yesterday, right?"
 <Page6>
-"I did say that, but... what? Are you still scared after all?"
-"No, that's not what I mean. If we're doing this as a team and not out of obligation, then I have a favor to ask of you.
+"I did say that, but... what? Did you get cold feet after all?"
+"No, that's not what I mean. If we're doing this as a team from now on, instead of out of obligation, then I have a favor to ask of you.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:19]: original line is better here imo

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 3]
```diff
@@ -27,37 +27,37 @@ That's why in return―――"
...
+"You think I could joke about something like this?
 I don't want to die either. So please listen to me. If you rest until tonight, could you defeat Vlov if you had the help of my eyes?"
 <Page10>
 Arcueid is the only one who can judge that.
 That's why... if she decides that it's possible, then I will wager everything on that possibility.
 <Page11>
 "...I see. If you move exactly the way I tell you to, then I guess we'd have the upper hand.
 But is that really okay? It would be much safer to just wait three days you know."
-"...I know that. But this is as much as I can handle. I can't wait any longer." // dropping mentally intentionally here because its awkward in english.
+"...I know that. But this is as much as I can handle. I can't bear to wait any longer." // dropping mentally intentionally here because its awkward in english.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:38]: not sure this is an improvement. It changes the register a bit in a way I don't really think fits with these two.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 4]
```diff
@@ -48,30 +48,30 @@ I won't be able to defeat Vlov with my strength tonight, but I can at least put
...
 <Page15>
 "I'm not going to do anything that uncivilized.
-It would be futile to hold him down directly. Besides, I've never done it before. It would be totally unnecessary."
+It would be futile to hold him down directly. Besides, I've never done it before, and I don't feel like starting now."
 <Page16>
 "I-I see. In that case, how are you going to subdue him?"
 "It depends on the location, but essentially I'll be using chains."
 <Page17>
-"Chains...? By chains, you mean those chains?"
+"Chains...? By chains, you mean those chains?"//He says あの鎖 but have we seen any chains to reference? Or is he just saying 'you mean, plain old chains'?
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:56]: I don't think the chains in question have been referenced or seen up to this point. I think he just means "Do you just mean normal chains" as in "is that all?"

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 5]
```diff
@@ -91,16 +91,16 @@ With that in mind, I look at Arcueid's face―――
...
-"You're going against a vampire from now on.
+"You're going up against a vampire from now on.
 That's why you must throw away your sense of morality, even if it's just for tonight. Carrying around that burden will only weigh you down when the time comes."
 <Page30>
 "...I know that much. Our opponent is a monster after all, which is why I'm willing to help."
 <Page31>
 "No, you don't understand at all, Shiki.
-Cutting off a limb? You should give up on such a suicidal act. If you have time to cut off his limbs, then you should start by cutting away his life force. It'd be different if you were any other creature, but this is something only you can do.
-Listen. Don't give Vlov a chance to fight back. The only advantage you have is that he doesn't know about your ability."
+Cutting off a limb? You should give up on such a suicidal act. If you have time to cut off his limbs, you should use that time to sever his life instead. It would be different if he were any other creature, but as it is this is something only you can do.//Is this the correct association of the last sentence? 他の生き物ならともかく、あなたにだけはそれが出来るんだから。
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:100]: I think this version makes more sense so I'm going to assume it's right. The only source of doubt is that the subject of the past couple of sentences is definitely Shiki.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 6]
```diff
@@ -139,11 +139,11 @@ At this rate, you will hesitate at the very last moment. And then you'll be kill
...
-"...I could take Vlov down with certainty if I just enchanted you with my Mystic Eyes. I'm not sure why, but even though this is the first time I've thought about doing it, I just can't bring myself to do it.
+"...I could take Vlov down with certainty if I just enchanted you with my Mystic Eyes. I'm not sure why, but even though this is the first time I've thought about doing it, I just can't bring myself to go through with it.
 It's like some sort of contradiction."
 <Page44>
 "I'll believe in you, Shiki. Let's defeat Vlov together."
 Arcueid turns around, returning from a gloomy to cheerful tone.
 <Page45>
 ...I wonder why.
-Even though her eyes were filled with trust, I can't help but feel extremely uneasy about that smile of hers.
+Even though her eyes were filled with trust, there was a terrible apprehension in her smile.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:149]: I can't help but sense a terrible apprehension in her smile.

**Review by Avuxo — APPROVED** [review]
few changes but mostly lgtm

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:19]: > ここからは義務じゃなくて協力になるんなら、俺の頼みも聞いてほしい。

Hmm, I wanted to capture the ここから / from now on to mark the change in their relationship. I guess I don't feel strongly about it / will revert.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3.txt [hunk 7]
```diff
@@ -1,36 +1,36 @@
 <Page1>
 "I see. In that case, I won't say anything.
-I'm glad that you decided to fight by my side. After all, we can easily 'kill' Vlov in one blow with you around."
+I'm glad that you've decided to fight by my side. After all, we can easily 'kill' Vlov in one blow with you around."
 <Page2>
 "...No, surely that's not possible.
-I can't even get close to him with how hot he makes the area.
-And besides, how are we going to find him?"
+He puts off so much heat that I can't even get close to him.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3.txt:6]: gives off so much heat

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3.txt [hunk 8]
```diff
@@ -1,36 +1,36 @@
...
 "I'm just more focused on efficiency.
 ...Well, whatever. Just so you know, it's going to be a war of attrition from here on out.
-The most reliable strategy until my strength returns to normal is to run.
-After three days, my <strength|L i f e  s c a l e> will be restored to about 30% of its original level. With that much power, I'll be able to defeat Vlov on my own."
+The safest strategy until my strength returns to normal is to run.
+After three days, my <strength|L i f e  S c a l e    > will be restored to about 30% of its original level. With that much power, I'll be able to defeat Vlov on my own."
 <Page5>
 "Three days...? But then..."
-"Yeah. During those three days, Vlov will be on the loose. He will go underground and use humans to replenish his blood supply... no, that's not all. From the looks of him, he might go out and roam the streets at night.
+"Yeah. During those three days, Vlov will be on the loose. He will go underground and use humans to replenish his blood supply... no, that's not all. From the looks of him, he might even go out and roam the streets at night.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3.txt:18]: あの様子からすると夜の街を徘徊するかもしれない

From the looks of **it**, he might even go out

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 9]
```diff
@@ -3,22 +3,22 @@
...
 <Page2>
-Us just going out looking for him and defeating him? That's not even worth considering.
-That kind of thinking is just an irresponsible escape from reality to ease my own guilt.
+Tracking him down and defeating him as we are right now? It's not even worth considering.
+That kind of wishful thinking is just an irresponsible escape from reality to ease my own guilt.
 <Page3>
 ...I need to collect myself.
-I must do so if I really want to defeat that vampire.
-I understand that if I truly want to reduce the number of casualties, I have no choice but to wait here.
+I have to if I really want to defeat that vampire.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:10]: > 本当にあの吸血鬼を倒したいのなら

Remove the 'I have to'.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 10]
```diff
@@ -3,22 +3,22 @@
...
 <Page4>
 But still―――three days is far too long.
 <Page5>
 "...Arcueid.
 You said that my duty as a guard ended yesterday, right?"
 <Page6>
-"I did say that, but... what? Are you still scared after all?"
-"No, that's not what I mean. If we're doing this as a team and not out of obligation, then I have a favor to ask of you.
+"I did say that, but... what? Did you get cold feet after all?"
+"No, that's not what I mean. If we're doing this as a team and not of out of obligation, then I have a favor to ask of you.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:19]: there shouldn't be an 'of' before out

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 11]
```diff
@@ -38,26 +38,26 @@ But is that really okay? It would be much safer to just wait three days you know
 "...I know that. But this is as much as I can handle. I can't wait any longer." // dropping mentally intentionally here because its awkward in english.
 <Page12>
 "Hmm... well, in that case, it's fine.
-And to be honest, there's only a small chance that we'll be able to avoid him for three days anyway."
+And to be honest, there's only a small chance that we'd be able to avoid him for three days anyway."
 <Page13>
-"So now I guess this is a strategy meeting.
+"So I guess from here on this is a strategy meeting.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:43]: comma after on

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt [hunk 12]
```diff
@@ -38,26 +38,26 @@ But is that really okay? It would be much safer to just wait three days you know
...
 <Page16>
 "...I see. Thank you, Arcueid."
 "I wouldn't recommend it though.
 You really are a mystery, Shiki."
 <Page17>
 I've increased my chances against Vlov.
-It's only natural that Arcueid is taken back; she's someone who values efficiency and certainty.
-But even so, Arcueid did not reject my idea.
+It's only natural that Arcueid can't understand; she's someone who values efficiency and certainty.
+But even so, Arcueid didn't veto my idea.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:60]: second Arc could be a she

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 13]
```diff
@@ -1,29 +1,29 @@
...
-After finishing a quick meal, I resume my conversation with Arcueid.
-At any rate, we've decided on a course of action. All that remains is to formulate a strategy.
+After finishing a light meal, I resume my conversation with Arcueid.
+The question of what we're going to do is settled. All that remains is to work out how to do it.
 The problem is those flames.# Until we do something about them, neither I nor Arcueid can get close to him.
 <Page5>
 "...I guess a surprise attack is our only option.
 It did seem like you were his target. We should have you act as a decoy and lure him out.
-Meanwhile, I'll hide in advance and attack him from behind during the window when he's headed toward you... or something like that."
+Meanwhile, I'll hide in advance and attack him from behind during the window where he's fixated on you... or something like that."
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:18]: when not where

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 14]
```diff
@@ -48,30 +48,30 @@ I won't be able to defeat Vlov with my strength tonight, but I can at least put
...
-...Once again, I come to realize the sheer outrageousness of Arcueid's being.
+...Once again, I am forced to contend with the sheer ludicrousness of Arcueid's existence.
 <Page22>
-But it's clear that if she can create those 'chains' at will, we'll be able to restrain Vlov for a short period of time.
+But it's true that if she can create those 'chains' at will, we should be able to restrain Vlov for a short period of time.
 <Page23>
 "...Well, anyway!
 I'll bind Vlov's limbs with those chains if I see an opening during our fight.
-I can restrain him for around ten seconds, so you'll have to come out with a swift, head-on attack.
+I can only fully restrain him for around ten seconds, so when I do you'll have to burst out from cover and come at him with a swift, head-on attack on his shortest line.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:74]: I think removing 'from cover and come at him' would be more concise

**Review by yk1512 — CHANGES_REQUESTED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt (hunk 9 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:10]: As in, just "I really want to defeat that vampire?" I feel like that loses the のなら

Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:10]: I was thinking of 'If I really want to defeat that vampire.' 
If you think that reads weirdly, then I'm ok with keeping the 'I have to.' Just wanted to try to be more faithful to the jp.

**Review by yk1512 — APPROVED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt (hunk 9 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2.txt:10]: Hm, yeah I think that in jp the のなら is a clearer backreference to the previous sentence and it helps to reify it in english by pulling the need to / have to into the line. Hewing closer to the JP I guess you could do a "If I really want to defeat that vampire, that is", but that feels stiff to me.

**Pull request merged**